src/components/FlavorStash/Note.js

```diff
@@ -0,0 +1,230 @@
+                  </Form.Group>
+                )}
+              </Field>
+            </Form.Row>
+            {values?.note ? (
+              <Form.Row>
+                <h4>Preview:</h4>
+                <Col md="12">
+                  {
+                    /* eslint-disable-next-line no-sync */
+                    unified()
+                      .use(parse)
+                      .use(remark2react)
+                      .processSync(values.note).result
+                  }
+                </Col>
+              </Form.Row>
+            ) : null}
+            <Field name="userId">
+              {({ input }) => <Form.Control {...input} type="hidden" />}
+            </Field>
+            <Field name="flavorId">
+              {({ input }) => <Form.Control {...input} type="hidden" />}
+            </Field>
+            <Field name="update">
+              {({ input }) => <Form.Control {...input} type="hidden" />}
+            </Field>
+            <Form.Row>
+              <Form.Group>
+                <ButtonGroup>
+                  <Button
+                    className="button-animation"
+                    variant="primary"
+                    type="submit"
+                    disabled={submitting}
+                  >
+                    <FontAwesomeIcon icon="save" size="sm" />
+                    &nbsp;<span>Save</span>
+                  </Button>
+                  <Button
+                    onClick={e => this.handleNoteEditor(false, e)}
+                    className="button-animation button--cancel"
+                  >
+                    <span>Cancel</span>
+                  </Button>
+                </ButtonGroup>
+              </Form.Group>
+            </Form.Row>
+          </Form>
+        )}
+      />
+    );
+  }
+
+  render() {
+    const { note, flavorId, loading } = this.props;
+    const { editingNote } = this.state;
+    const updatedNote = this.state.updatedNote;
+
+    if (loading !== flavorId) {
+      return (
+        <Fragment>
+          <h3>
+            {note[flavorId] || updatedNote
+              ? this.editNoteIcon(flavorId)
+              : this.addNoteIcon(flavorId)}{' '}
+            Flavor Notes
+          </h3>
+          {!editingNote && (note[flavorId] || updatedNote) ? (
+            <Fragment>
+              {
+                /* eslint-disable-next-line no-sync */
+                unified()
+                  .use(parse)
+                  .use(remark2react)
+                  .processSync(updatedNote || note[flavorId].note).result
+              }
+            </Fragment>
+          ) : null}
+          {editingNote ? this.noteEditor() : null}
+        </Fragment>
+      );
+    } else {
+      return <Fragment>Loading...</Fragment>;
+    }
+  }
+}
+
+export const mapStateToProps = state => ({
+  loading: isLoading(state),
+  note: getFlavorNote(state)
+});
+
+export const mapDispatchToProps = dispatch => ({
+  actions: bindActionCreators(
+    {
+      ...noteActions
```

Comment:
No need to spread if it's the only action group being used.

Reply:
Also fixed in the page as well.